ObjectiveGit/GTRepository.h

```diff
@@ -228,6 +203,9 @@ extern NSString * const GTRepositoryInitOptionsOriginURLString;
 ///                         May be NULL.
 ///
 /// returns nil (and fills the error parameter) if an error occurred, or a GTRepository object if successful.
++ (nullable instancetype)cloneFromURL:(NSURL *)originURL toWorkingDirectory:(NSURL *)workdirURL options:(nullable NSDictionary *)options error:(NSError **)error transferProgressBlock:(nullable void (^)(const git_transfer_progress *, BOOL *stop))transferProgressBlock;
+
+/// Backward-compatible method that uses `GTRepositoryCloneOptionsCheckout`
```

Comment:
I actually started by keeping the old methods "checkout-ing" methods in (this one and the `-checkoutStuff:` below). Then it started to get messy with keeping the old "no notify" methods, so they were slashed, but this one stayed in. If you don't mind about bc, I can :fire: it.